src/node_options.cc

```diff
@@ -274,6 +274,10 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             "experimental ES Module support and caching modules",
             &EnvironmentOptions::experimental_modules,
             kAllowedInEnvironment);
+  AddOption("--experimental-wasm-modules",
+            "experimental ES Module support for webassembly modules",
+            &EnvironmentOptions::experimental_wasm_modules,
+            kAllowedInEnvironment);
```

Comment:
Please add a check that the flag is passed with `--experimental-modules`. See https://github.com/nodejs/node/blob/a9dba75d8f166c54d489a0926ba86ef632152367/src/node_options.cc#L120-L124